src/Uno.UI.Runtime.Skia.Linux.FrameBuffer/FramebufferHost.cs

```diff
@@ -101,7 +102,7 @@ private void Initialize()
 			ApiExtensibility.Register(typeof(Windows.UI.ViewManagement.IApplicationViewExtension), o => new ApplicationViewExtension(o));
 			ApiExtensibility.Register(typeof(Windows.Graphics.Display.IDisplayInformationExtension), o => _displayInformationExtension ??= new DisplayInformationExtension(o, DisplayScale));
 
-			void Dispatch(System.Action d)
+			void Dispatch(System.Action d, NativeDispatcherPriority p)
```

Comment:
As the priority not used here, could it be problematic?

Reply:
@MartinZikmund It has to stay, unfortunately, as all platforms should have the same signature.

Reply:
I rather meant whether in FrameBuffer it could cause problems when the Measure/Arrange is always planned immediately - whether it could not block other UI operations with "higher priority" potentially?

Reply:
@MartinZikmund I don't see why it would be problematic?

Internally, `NativeDispatcher` will respect the priority by dispatching from the highest non-empty priority queue.

It was only needed on Wpf where we are dispatching using Wpf's dispatcher which also has a priority.

Reply:
I see!